tests/test_workers/test_tasks/test_build_merge_index_image.py

```diff
@@ -397,6 +397,7 @@ def test_add_bundles_missing_in_source(
         1,
         'amd64',
         '4.6',
+        'private',
```

Comment:
distribution_scope is usually `stage`, `prod` or `dev`

Reply:
'Private' replaced for 'dev'. Is that okay?